MODULE.bazel

```diff
@@ -0,0 +1,57 @@
+module(
+    name = "rules_fuzzing",
+    # Automatically updated during releases by the Publish to BCR app.
+    version = "0.0.0",
+)
+
+bazel_dep(name = "abseil-cpp", version = "20230802.1", repo_name = "com_google_absl")
```

Comment:
Abseil has been upgraded since. Does it work with:
```
bazel_dep(name = "abseil-cpp", version = "20240116.0")
```
?

Reply:
Since Bzlmod uses minimum version selection, rulesets should ideally specify the lowest version they work with to allow users the greatest flexibility. If a higher version of abseil at the same `compatibility_level` breaks rules_fuzzing, that would be considered a bug in the abseil module.